src/main/java/tel/schich/javacan/BcmMessage.java

```diff
@@ -164,32 +198,53 @@ public BcmOpcode getOpcode() {
     }
 
     /**
-     * Returns the count for ival1 repetitions of this message.
+     * Returns the count for {@link #getInterval1()} repetitions of this message.
      */
     public int getCount() {
         return buffer.getInt(base + OFFSET_COUNT);
     }
 
     /**
-     * Returns ival1 from this message.
+     * The {@code interval1} has different meanings depending on the {@link #getOpcode()} of the
+     * message:
+     * <ul>
+     * <li><b>When used with {@link BcmOpcode#TX_SETUP}:</b></br>
+     * The broadcast manager sends {@link #getCount()} messages with this interval, then continue to
+     * send at {@link #getInterval2()}. If only one timer is needed set
+     * {@link BcmMessageBuilder#count(int) count} to {@code 0} and {@link BcmMessageBuilder#ival1
+     * interval1} to {@code null}.</li>
+     * <li><b>When used with {@link BcmOpcode#RX_SETUP}:</b></br>
+     * Send {@link BcmOpcode#RX_TIMEOUT} when a received message is not received again within the given
+     * interval. When {@link BcmFlag#STARTTIMER} is set, the timeout detection is activated directly -
+     * even without a former CAN frame reception.</li>
+     * </ul>
      *
-     * @return the {@code ival1} value or {@code null} if it is not set
+     * @return the duration or {@code null} if it is not set
      */
-    public BcmTimeval getIval1() {
+    public Duration getInterval1() {
         long sec = getPlatformLong(base + OFFSET_IVAL1_TV_SEC);
         long usec = getPlatformLong(base + OFFSET_IVAL1_TV_USEC);
-        return (sec + usec) != 0 ? new BcmTimeval(sec, usec) : null;
+        return (sec + usec) != 0 ? Duration.ofSeconds(sec).plusMillis(usec) : null;
```

Comment:
This should probably be `plusNanos(usec * 1000)` (same below)

Reply:
Also instead of null we could also return [ZERO](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#ZERO)

Reply:
Okay, this might save some NPEs in the future.

Reply:
> This should probably be `plusNanos(usec * 1000)` (same below)

Why should i introduce a magic number `1000` if the API can handle the micro-seconds?

Reply:
> This should probably be `plusNanos(usec * 1000)` (same below)

ah, crap... i read `micro`. The brain reads what it wants to read.

Reply:
that's why we do code review :)